tests/Umbraco.Tests.Integration/ManagementApi/Dictionary/ByKeyDictionaryControllerTests.cs

```diff
@@ -0,0 +1,49 @@
+using System.Linq.Expressions;
+using System.Net;
+using NUnit.Framework;
+using Umbraco.Cms.Api.Management.Controllers.Dictionary;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Tests.Integration.ManagementApi.Dictionary;
+
+[TestFixture]
+public class ByKeyDictionaryControllerTests : DictionaryBaseTest<ByKeyDictionaryController>
+{
+    private Guid _dictionaryId;
+
+    protected override Expression<Func<ByKeyDictionaryController, object>> MethodSelector =>
+        x => x.ByKey(_dictionaryId);
+
+    [SetUp]
+    public void Setup() => _dictionaryId = CreateDictionaryItem("TestDictionaryItem");
+
+    protected override UserGroupAssertionModel AdminUserGroupAssertionModel => new()
+    {
+        ExpectedStatusCode = HttpStatusCode.OK
```

Comment:
If the changes above are applied these StatusCode returned would probably be changed to NotFound